.nuspec/Xamarin.Forms.targets

```diff
@@ -183,7 +183,39 @@
     <Error Code="XF005"  Condition="$(TargetFrameworkVersionWithoutV) &lt; $(MinTargetFrameworkVersionForForms)"
            Text="The %24(TargetFrameworkVersion) for $(ProjectName) ($(TargetFrameworkVersion)) is less than the minimum required %24(TargetFrameworkVersion) for Xamarin.Forms ($(MinTargetFrameworkVersionForForms)). You need to increase the %24(TargetFrameworkVersion) for $(ProjectName)."   />
   </Target>
+
+
+  <!-- Xamarin iOS targets -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == 'Xamarin.iOS'">
+    <CoreCompileDependsOn>
+      IncludeCorrectXIOSReference;
+      $(CoreCompileDependsOn);
+    </CoreCompileDependsOn>
+    <PrepareForBuildDependsOn>
+      IncludeCorrectXIOSReference;
+      $(PrepareForBuildDependsOn);
+    </PrepareForBuildDependsOn>
+  </PropertyGroup>
+
   
+  <Target Name="IncludeCorrectXIOSReference" Condition="'$(TargetFrameworkIdentifier)' == 'Xamarin.iOS'">
+	<PropertyGroup>
+		<Use2017 Condition="'$(Use2017)' == '' AND '$(MSBuildRuntimeType)' == 'Mono'">$(FrameworkSDKRoot.Contains('/Versions/5'))</Use2017>
+		<Use2017 Condition="'$(Use2017)' == '' AND '$(MSBuildAssemblyVersion)' &lt; '16.0'">true</Use2017>
+		<Use2017 Condition="'$(Use2017)' == ''">false</Use2017>
+	</PropertyGroup>
+    <ItemGroup Condition="'$(Use2017)' != 'true'">
```

Comment:
Use positive condition (== 'false' is easier to read)

Reply:
But that's not the equivalent check. The equivalent check would be

```XAML
<ItemGroup Condition="'$(Use2017)' == 'false' OR "'$(Use2017)' == ''>
```

it's good to use != anywhere that you want to articulate the default path. That way at some point if Use2017 goes away this path will still run because it represents the default

Line 205 is probably kind of the excessive one but I don't really care. It's easier to have it set to something so when I inspect the binlogs I can validate that as well